packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.tsx

```diff
@@ -53,17 +53,19 @@ const TransactionStatus: React.FC<TTransactionStatus> = ({ transactionType }) =>
     return (
         <div className='wallets-transaction-status'>
             <div className='wallets-transaction-status__header'>
-                <WalletText weight='bold'>Transaction status</WalletText>
+                <WalletText size='sm' weight='bold'>
+                    Transaction status
+                </WalletText>
                 {isError && <Warning />}
             </div>
             <Divider color='#d6dadb' /> {/* --color-grey-5 */}
             <div className='wallets-transaction-status__body'>
                 {!isError && isLoading && <WalletsTransactionStatusLoader />}
                 {isError && <TransactionStatusError refresh={refresh} />}
-                {!isLoading && !isError && wallet && transactions && (
```

Comment:
This change was also made because when there the data is fetched (`isLoading=false` and `isError=false`) no transactions are returned as `undefined` rather than empty list from the hook